utils/s2n_random.h

```diff
@@ -28,6 +28,7 @@ struct s2n_rand_device {
     dev_t rdev;
 };
 
+bool s2n_supports_custom_rand();
```

Comment:
```suggestion
bool s2n_supports_custom_rand(void);
```

Reply:
Fixed this and opened an issue for others in our codebase https://github.com/aws/s2n-tls/issues/4952